plugins/notifications-backend/src/service/router.ts

```diff
@@ -177,7 +177,7 @@ export async function createRouter(
     response.json({ status: 'ok' });
   });
 
-  router.get('/', async (req, res) => {
+  router.get('/notifications', async (req, res) => {
```

Comment:
By doing this, we will end up with URLs like `https://host.com/notifications/notifications`.

I do not see another resource to be added/considered under the top-level `/notifications/` context in the future.
For this reason, I would stick to just a single `/notifications` in the URL.

This is related to: https://github.com/backstage/backstage/pull/23645/files#diff-e2b0532e3202f4ab3b59bf9abd868e81fbe500bddf84148c467487447eb1b420R239

We just need to make sure that the `/:id` handler is the last one.

Reply:
We will have settings later on here as well